ompi/attribute/attribute.h

```diff
@@ -155,17 +136,17 @@ typedef union ompi_attribute_fortran_ptr_t ompi_attribute_fortran_ptr_t;
 
 struct ompi_attribute_keyval_t {
     opal_object_t super;
-    ompi_attribute_type_t attr_type; /**< One of COMM/WIN/DTYPE. This
-                                       will be used to cast the
-                                       copy/delete attribute functions
-                                       properly and error checking */
-    int attr_flag; /**< flag field: contains "OMPI_KEYVAL_PREDEFINED",
-                      "OMPI_KEYVAL_F77"  */
+    ompi_attribute_type_t attr_type;            /**< One of COMM/WIN/DTYPE. This
+                                                  will be used to cast the
+                                                  copy/delete attribute functions
+                                                  properly and error checking */
+    int attr_flag;                              /**< flag field: contains "OMPI_KEYVAL_PREDEFINED",
+                                                   "OMPI_KEYVAL_F77"  */
     ompi_attribute_fn_ptr_union_t copy_attr_fn; /**< Copy function for the
                                              attribute */
     ompi_attribute_fn_ptr_union_t delete_attr_fn; /**< Delete function for the
                                                attribute */
-    ompi_attribute_fortran_ptr_t extra_state; /**< Extra state of the attribute */
+    ompi_attribute_fortran_ptr_t extra_state;     /**< Extra state of the attribute */
```

Comment:
Does it interpret `/**` as meaning, please put some space here?

Reply:
It is aligning the comments. We really should avoid the /**<**/ doxygen comments and just put the comment above the strict member.